website/docs/r/dedicated_hardware_security_module.html.markdown

```diff
@@ -66,7 +66,8 @@ resource "azurerm_public_ip" "example" {
   name                = "example-pip"
   location            = azurerm_resource_group.example.location
   resource_group_name = azurerm_resource_group.example.name
-  allocation_method   = "Dynamic"
+  allocation_method   = "Static"
+  sku                 = "Standard"
```

Comment:
The sku for public IP is `Standard` in 4.0 by default, so this doesn't need to be specified in v4.0+
```suggestion
```

Reply:
Provider documentation is versioned on the registry 
![image](https://github.com/user-attachments/assets/820bb730-f75d-418b-b8f8-a217d0c1764a)

And this change will not apply to 3.x versions of the documentation, only 4.x going forward.